x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -259,7 +259,8 @@ export const EditExceptionModal = memo(function EditExceptionModal({
             <ModalBodySection>
               <EuiFormRow fullWidth>
                 <EuiCheckbox
-                  id="close-alert-on-add-add-exception-checkbox"
+                  data-test-subj="close-alert-on-add-edit-exception-checkbox"
+                  id="close-alert-on-add-edit-exception-checkbox"
```

Comment:
I don't think you need this id here do you? Looks like above you're using data-test-subj successfully?

Reply:
i believe the `id` is a required field for the `EuiCheckbox`, would spit out a console warning and potentially cause usage problems if not there.

Reply:
Yeah looking at the docs and searching around this looks like it is required.